plugins/sdm/lib/role_grant_helper.py

```diff
@@ -34,4 +34,4 @@ def __allowed_to_assign_role(self, role_name, sdm_account):
         if not self.__bot.config['USER_ROLES_TAG']:
             return True
         permitted_roles = sdm_account.tags.get(self.__bot.config['USER_ROLES_TAG']) if sdm_account.tags else ""
```

Comment:
I'll approve the PR, but please fix change this line in future versions. Based on the return statement below, I think this line should be:
```
permitted_roles = sdm_account.tags.get(self.__bot.config['USER_ROLES_TAG']) if sdm_account.tags else None
```